core/controller.go

```diff
@@ -38,12 +41,20 @@ func (c *Controller) Activate() *resources.ActivateResponse {
 	return &resources.ActivateResponse{Implements: []string{"VolumeDriver"}}
 }
 
-func (c *Controller) Create(createRequest *resources.CreateRequest) *resources.GenericResponse {
+func (c *Controller) Create(createVolumeRequest *resources.CreateVolumeRequest) *resources.GenericResponse {
```

Comment:
should be:
func (c *Controller) Create(createVolumeRequest resources.CreateVolumeRequest) resources.GenericResponse